spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -1542,14 +1542,15 @@ def get_current_shellwidget(self):
     @Slot(bool, bool)
     @Slot(bool, str, bool)
     def create_new_client(self, give_focus=True, filename='', is_cython=False,
-                          is_pylab=False, is_sympy=False, given_name=None):
+                          is_pylab=False, is_sympy=False, given_name=None,
+                          no_cache=False):
```

Comment:
Let's change this to be `cache=True` because it's easier to understand than a double negative.

In other words, I think to reason about this it's easier to ask

> is `cache` True?

than

> is `no_cache` not True?